packages/perseus/src/widgets/radio/__tests__/answerless-radio.test.tsx

```diff
@@ -0,0 +1,177 @@
+                correct: true,
+            },
+            {
+                content: "Correct 2",
+                correct: true,
+            },
+            {
+                content: "Incorrect",
+                correct: false,
+            },
+            {
+                content: "None",
+                isNoneOfTheAbove: true,
+                correct: false,
+            },
+        ],
+
+        numCorrect: 2,
+
+        /*
+        Radio derives `numCorrect` from answers
+        and uses `numCorrect` for instruction text if both:
+        1. multipleSelect is true
+        2. countChoices is true
+
+        Since answers can't continue to be on the FE,
+        we need to assert that this functionality continues
+        to work without answers
+        */
+        multipleSelect: true,
+        countChoices: true,
+
+        hasNoneOfTheAbove: false,
+        randomize: false,
+        deselectEnabled: false,
+        onePerLine: false,
+        displayCount: null,
+        noneOfTheAbove: false,
+    };
+
+    return {
+        ...base,
+        ...modifier,
+    };
+}
+
+describe("interactive: full vs answerless", () => {
+    beforeAll(() => {
+        registerAllWidgetsForTesting();
+    });
+
+    let userEvent: UserEvent;
+    beforeEach(() => {
+        userEvent = userEventLib.setup({
+            advanceTimers: jest.advanceTimersByTime,
+        });
+
+        jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
+            testDependencies,
+        );
+
+        // Mocked for loading graphie in svg-image
+        global.fetch = jest.fn(() =>
+            Promise.resolve({
+                text: () => "",
+                ok: true,
+            }),
+        ) as jest.Mock;
+    });
+
+    it("is interactive with full widget options", async () => {
+        // Arrange
+        const fullWidgetOptions: PerseusRadioWidgetOptions =
+            getRadioWidgetOptions();
+
+        const fullItem: PerseusRenderer = {
+            content: "[[☃ radio 1]]",
+            images: {},
+            widgets: {
+                "radio 1": {
+                    type: "radio",
+                    options: fullWidgetOptions,
+                },
+            },
+        };
+
+        // Act
+        const {renderer} = renderQuestion(fullItem);
+
+        await userEvent.click(
+            screen.getByRole("checkbox", {name: "(Choice A) Correct 1"}),
+        );
+        await userEvent.click(
+            screen.getByRole("checkbox", {name: "(Choice B) Correct 2"}),
+        );
+
+        // assert that functionality previous based on answers still works
+        expect(screen.getByRole("group", {name: "Choose 2 answers:"}));
+
+        const userInput = renderer.getUserInputMap();
+        const score = scorePerseusItem(fullItem, userInput, "en");
+
+        // Assert
+        expect(score).toHaveBeenAnsweredCorrectly();
+    });
+
+    it("is interactive with stripped widget options", async () => {
```

Comment:
We're using a few different terms for this concept:

- public options
- stripped options
- answerless options

I'd prefer to standardize on one, for the sake of future maintainers.

Reply:
The functions are all named `getPublicWidgetOptions`... so it feels like we've gone with "public".  I'm a bit averse to "stripped" after starting with it. It feels aggressive. "Answerless" isn't always 100% correct because there are situations where we remove data that isn't the answer, but could reveal info about the answer. 🤷‍♂️

Reply:
I personally think "public" is the most incorrect. Answerless/answerful are both public, just at different times. But I can change it for this PR.

Reply:
+1 for keeping things consistent for now. A temporarily unclear name is better than temporary inconsistency. Find-replace is easy, but gets harder when naming isn't consistent.